src/components/bip-action-card/bip-action-card.spec.ts

```diff
@@ -12,7 +12,7 @@ describe('BipActionCardComponent', function() {
 
   beforeEach(async () => {
     await TestBed.configureTestingModule({
-      declarations: [ BipActionCardComponent ]
+      declarations: [BipActionCardComponent]
```

Comment:
```suggestion
      declarations: [ BipActionCardComponent ]
```

Reply:
This change is not consistant with the TSLint rules.
https://github.com/MediaComem/biopocket-mobile/blob/sprint-2-introducing-actions/tslint.json#L10